code/modules/reagents/reagent_containers/food/drinks/champagne.dm

```diff
@@ -0,0 +1,57 @@
+/decl/material/liquid/ethanol/champagne
+	name = "Champagne"
+	lore_text = "Sparkling wine made from exquisite grape."
+	taste_description = "bitter taste"
+	color = "#a89410"
+	strength = 18
+
+	glass_name = "champagne"
+	glass_desc = "Wow, it's bubbling!"
+	glass_special = list(DRINK_FIZZ)
+
+/obj/effect/decal/cleanable/champagne
+	name = "champagne"
+	desc = "Is this a gold?"
+	gender = PLURAL
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "fchampagne1"
+	color = COLOR_BRASS
+	random_icon_states = list("fchampagne1", "fchampagne2", "fchampagne3", "fchampagne4")
+
+/obj/item/chems/food/drinks/bottle/champagne
+	name = "\improper Champagne bottle"
```

Comment:
Champagne is only a proper noun with a capital if you're talking about the region of france, otherwise it's just sparkling lowercase.